statstool

```diff
@@ -35,18 +35,16 @@ set -eu -o pipefail
     }
 }
 
-PR_SAMPLESIZE=250
-COMMIT_PR_SAMPLESIZE=200
+PR_SAMPLESIZE=2500
```

Comment:
I think 250/200 is fine for these samples. The problem with increasing this is that each request is one API request done serially. if you increase this 10-fold I bet you will see the crawler running _much_ slower. And this sample is really only there to estimate if our heuristic link between commits and PRs (merge sha in PR + autolink patterns in commit messages) is good. Having 250/200 is sufficient for that.

Reply:
I will revert to 250/200.